.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
```

Comment:
If you rely on a babelrc config then you might want to remove the config we pass to babel-core/register